src/rpcblockchain.cpp

```diff
@@ -759,6 +774,80 @@ UniValue beaconreport(const UniValue& params, bool fHelp)
     return results;
 }
 
+UniValue verifiedbeaconreport(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() > 0)
+        throw runtime_error(
+                "verifiedbeaconreport\n"
+                "\n"
+                "Displays list of verified beacons in the network\n");
+
+    UniValue results(UniValue::VOBJ);
+
+    std::vector<std::pair<NN::Cpid, NN::Beacon>> active_beacons;
+
+    UniValue verified_from_global(UniValue::VARR);
+
+    const ScraperPendingBeaconMap& verified_beacons_from_global = GetVerifiedBeaconsForReport(true);
+
+    for (const auto&verified_beacon_pair : verified_beacons_from_global)
+    {
+        UniValue entry(UniValue::VOBJ);
+
+        entry.pushKV("cpid", verified_beacon_pair.first);
+        entry.pushKV("timestamp", verified_beacon_pair.second.timestamp);
+
+        verified_from_global.push_back(entry);
+    }
+
+    results.pushKV("verified beacons from scraper global", verified_from_global);
+
+
+    UniValue verified_from_convergence(UniValue::VARR);
+
+    const ScraperPendingBeaconMap& verified_beacons_from_convergence = GetVerifiedBeaconsForReport(false);
```

Comment:
Did you mean to declare the `verified_beacons_from_convergence` variable as a reference? I'm not sure if C++ will always let the temporary object live until the function finishes using it.

Reply:
Nope. I should fix that.